FindTown/FindTown/Presentation/FavoriteScene/Favorite/Favorite1ViewController.swift

```diff
@@ -0,0 +1,132 @@
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Functions
+    
+    override func setupView() {
+        self.title = "찜"
+        self.view.backgroundColor = FindTownColor.back2.color
+    }
+    
+    override func addView() {
+        super.addView()
+        
+        [anonymousView, isEmptyView, favoriteTableView].forEach {
+            self.view.addSubview($0)
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+        
+        let view = self.view.safeAreaLayoutGuide
+
+        NSLayoutConstraint.activate([
+            anonymousView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            anonymousView.topAnchor.constraint(equalTo: view.topAnchor),
+            anonymousView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            anonymousView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+        ])
+        
+        NSLayoutConstraint.activate([
+            isEmptyView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            isEmptyView.topAnchor.constraint(equalTo: view.topAnchor),
+            isEmptyView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            isEmptyView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+        ])
+        
+        NSLayoutConstraint.activate([
+            favoriteTableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            favoriteTableView.topAnchor.constraint(equalTo: view.topAnchor),
+            favoriteTableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            favoriteTableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+        ])
+    }
+    
+    override func bindViewModel() {
+        
+        // MARK: ViewModel - Input
+        
+        self.anonymousView.signUpButton.rx.tap
+            .bind { [weak self] in
+                self?.viewModel?.input.signUpButtonTrigger.onNext(())
+            }
+            .disposed(by: disposeBag)
+        
+        self.isEmptyView.findOutButton.rx.tap
+            .bind { [weak self] in
+                /// 홈 탭으로 이동
+                self?.tabBarController?.selectedIndex = 0
+            }
+            .disposed(by: disposeBag)
+        
+        // MARK: ViewModel - Output
+        
+        self.viewModel?.output.viewStatus
+            .bind(to: self.rx.favoriteViewStatus)
+            .disposed(by: disposeBag)
+        
+        self.viewModel?.output.favoriteDataSource
+            .observe(on: MainScheduler.instance)
+            .filter { !$0.isEmpty }
+            .bind(to: favoriteTableView.rx.items(cellIdentifier: TempTableViewCell.reuseIdentifier,
+                                                 cellType: TempTableViewCell.self)) {
+                index, item, cell in
+                cell.textLabel?.text = item.name
```

Comment:
```suggestion
                // sepupCell()
```

아마 지금은 텍스트밖에 없어서 이렇게 하신거 같지만..! 나중에 이런 메서드 하나 만드셔도 괜찮아 보여요 ㅎㅎㅎ 😄

Reply:
아아 맞아요 ! 다만 나중에 cell 머지 받으면 cell 안에 메서드를 만드는게 어떨까 싶어서 아직 구현하지 않았습니다 ㅎㅎ !